src/templates/base/2019/base.html

```diff
@@ -278,6 +289,80 @@
           });
         }
 
+        function closeAnyOtherOpenDropdown(e) {
+          if (e.target.classList.contains('dropdown-open')) {
+            return
+          };
+          var openDropdownBtn = document.querySelector('.nav-dropdown-btn.dropdown-open');
+          openDropdownBtn && openDropdownBtn.click();
+        }
+        function trapFocusInList(e) {
+          var list = e.currentTarget;
+          var isInFooter = list.classList.contains('footer-list');
+          if(e.key === "ArrowDown") {
```

Comment:
In the language selector in the footer, the "Down" key selects the Spanish option first. Pressing "Down" again selects "Help Translate" next.

In order to traverse all of the language options in order, we need to press Down to select Spanish, then Up, Up, Up.

For the footer list options, which are reversed, I think pressing "Down" at first should select the last option ("Help Translate") and the "Up" key should select the first option ("Spanish").

Reply:
This is something that i had thought about initially but wasn't sure if i should add that functionality because even in footer the chevron icon still points down but because visually the list was longer than available space we've decided to render list above button. so because of that i was unsure if we should just support arrow down key or arrow up as well.
Now that you've also noticed the lack of it makes me believe thats how people might expect it to work, so added that logic as well

Reply:
LGTM now!